src/SUMMARY.md

```diff
@@ -353,46 +353,50 @@
-  - [Scoped Threads](concurrency/scoped-threads.md)
+  - [Plain Threads](concurrency/threads/threads.md)
+  - [Scoped Threads](concurrency/threads/scoped.md)
 - [Channels](concurrency/channels.md)
+  - [Senders and Reveivers](concurrency/channels/senders-receivers.md)
   - [Unbounded Channels](concurrency/channels/unbounded.md)
   - [Bounded Channels](concurrency/channels/bounded.md)
 - [`Send` and `Sync`](concurrency/send-sync.md)
+  - [Marker Traits](concurrency/send-sync/marker-traits.md)
   - [`Send`](concurrency/send-sync/send.md)
   - [`Sync`](concurrency/send-sync/sync.md)
   - [Examples](concurrency/send-sync/examples.md)
-- [Shared State](concurrency/shared_state.md)
-  - [`Arc`](concurrency/shared_state/arc.md)
-  - [`Mutex`](concurrency/shared_state/mutex.md)
-  - [Example](concurrency/shared_state/example.md)
-- [Exercises](exercises/concurrency/morning.md)
-  - [Dining Philosophers](exercises/concurrency/dining-philosophers.md)
-  - [Multi-threaded Link Checker](exercises/concurrency/link-checker.md)
-  - [Solutions](exercises/concurrency/solutions-morning.md)
+- [Shared State](concurrency/shared-state.md)
+  - [`Arc`](concurrency/shared-state/arc.md)
+  - [`Mutex`](concurrency/shared-state/mutex.md)
+  - [Example](concurrency/shared-state/example.md)
+- [Exercises](concurrency/sync-exercises.md)
+  - [Dining Philosophers](concurrency/sync-exercises/dining-philosophers.md)
+  - [Multi-threaded Link Checker](concurrency/sync-exercises/link-checker.md)
+  - [Solutions](concurrency/sync-exercises/solutions.md)
 
 # Concurrency: Afternoon
 
-- [Async Basics](async.md)
-  - [`async`/`await`](async/async-await.md)
-  - [Futures](async/futures.md)
-  - [Runtimes](async/runtimes.md)
-    - [Tokio](async/runtimes/tokio.md)
-  - [Tasks](async/tasks.md)
-  - [Async Channels](async/channels.md)
-- [Control Flow](async/control-flow.md)
-  - [Join](async/control-flow/join.md)
-  - [Select](async/control-flow/select.md)
-- [Pitfalls](async/pitfalls.md)
-  - [Blocking the Executor](async/pitfalls/blocking-executor.md)
-  - [`Pin`](async/pitfalls/pin.md)
-  - [Async Traits](async/pitfalls/async-traits.md)
-  - [Cancellation](async/pitfalls/cancellation.md)
-- [Exercises](exercises/concurrency/afternoon.md)
-  - [Dining Philosophers](exercises/concurrency/dining-philosophers-async.md)
-  - [Broadcast Chat Application](exercises/concurrency/chat-app.md)
-  - [Solutions](exercises/concurrency/solutions-afternoon.md)
+- [Welcome](concurrency/welcome-async.md)
+- [Async Basics](concurrency/async.md)
+  - [`async`/`await`](concurrency/async/async-await.md)
+  - [Futures](concurrency/async/futures.md)
+  - [Runtimes](concurrency/async/runtimes.md)
+    - [Tokio](concurrency/async/runtimes/tokio.md)
+  - [Tasks](concurrency/async/tasks.md)
+- [Channels and Control Flow](concurrency/async/control-flow.md)
+  - [Async Channels](concurrency/control-flow/channels.md)
+  - [Join](concurrency/control-flow/join.md)
+  - [Select](concurrency/control-flow/select.md)
```

Comment:
I somehow feel there should be an `async` in the URLs here to highlight that these slides are specific to that sub-topic. What do you think of calling this segment "Async Channels and Control Flow"? The folder would then be `concurrency/async-control-flow/`.

Reply:
That title is too long (won't fit in the sidebar), but I will change the folder name.

Reply:
Thanks, that sounds good!